js_modules/dagster-ui/packages/ui-core/src/search/types.ts

```diff
@@ -14,12 +14,33 @@ export enum SearchResultType {
   Resource,
 }
 
+export enum AssetFilterSearchResultType {
+  // Add types with corresponding strings to distinguish
+  // between SearchResultType.AssetGroup
+  ComputeKind = 'AssetFilterSearchResultType.ComputeKind',
+  CodeLocation = 'AssetFilterSearchResultType.CodeLocation',
+  Owner = 'AssetFilterSearchResultType.Owner',
+  AssetGroup = 'AssetFilterSearchResultType.AssetGroup',
+}
+
+export function isAssetFilterSearchResultType(
```

Comment:
Not sure how to check if a value exists in a given enum... I was seeing the following always return true:
```
x: SearchResultType | AssetFilterSearchResultType = ...
x in AssetFilterSearchResultType // Returns true??
```
For now, just tested for strict equivalence to make this logic work

Reply:
Got it, thanks.

I think we still need this logic within the asset search component to render asset results and filter results a little differently.

Reply:
I moved this function to the other PR where it's actually used